Sharing and using the forum (#15)
tiosgz/modding-vivaldi: community/asking-questions.md

> # Asking Questions on the Forum

Reviewer: Maybe also a section about asking for complete mods? E.g. describe thoroughly, include mockups…

Author: Ok.

Author(s): code3z (@code3 on the forum)

> The [modding forum](https://forum.vivaldi.net/category/52/modifications) is also a great place to ask questions about modding. Before asking, search the forum and the internet to make sure there is not already a good answer.

Reviewer: ~~make sure~~ → check if

(‘make sure’ sounds like you’ll kill them if there’s an answer *somewhere*)

Author: > ‘make sure’ sounds like you’ll kill them if there’s an answer somewhere

I mean, I never said I wouldn’t 🤷‍♂️

I’ll fix that.

When asking a question, keep a few simple things in mind:
 - Use a topic title that is descriptive
 - Be polite and don't use ALL CAPS

>  - If you are asking questions or trying to debug a certain piece of code, share that code

Reviewer: > If you are asking questions or trying to debug a certain piece of code

‘if you are asking about a certain piece of code’?

Author: Sure.

# Asking Questions on the Forum

Author(s): code3z (@code3 on the forum)

The [modding forum](https://forum.vivaldi.net/category/52/modifications)

> is also a great place to ask questions about modding. Before asking, search the

Reviewer: ‘also’ like what? Did you leave it here by accident?

forum and the internet to check if there is not already a good answer.

When asking a question, keep a few simple things in mind:
 - Use a topic title that is descriptive
 - Be polite and don't use ALL CAPS
 - If you are asking questions about a certain piece of code, share that code

>  - Share any ways you thought you might fix the problem or things that didn't work

Reviewer: ```suggestion
 - Share any ways you thought might fix the problem even if they didn’t work
```

>  - Don't add the "mod" tag to your post - save that for when you actually publish a mod

Reviewer: I’ll ask someone if a `modding question` tag would make sense.

Author: Yeah, I don't think `question` or `modding question` exist, but I guess we can create them.

Reviewer: Our dear Dragon has reminded me about question topics (select ‘ask as question’ in the *submit* button drop-down). I think that’s enough if the topics are in the modding category, what do you think?

Author: Sure, I’ll add that.

Author: Uh-oh. I no longer see the "add tags" box on the modding forum. Do you?

> ## Asking For mods

Reviewer: ```suggestion
## Asking for Mods
```
(yes, I’ll go with the standard rules – but this didn’t follow even them)

Please check the other headings as well.

> You can use the forum to ask someone to make a mod for you.

Reviewer: Early newline if I’m not wrong.

But remember that it is a big favor: if they wanted this mod themselves,
they would have almost certainly made it. So be patient, describe what you're
looking for clearly, and realize that you may not get an answer at all
(especially if what you're asking is hard to do).

If you think that what you're asking should be implemented in Vivaldi directly,

> remember to make a feature request for it in the correct section of the forum.

Reviewer: Include a link?
